BE/LegalEntities/NorthAmericanEntities/USExampleEntities.rdf

```diff
@@ -0,0 +1,270 @@
+		<fibo-be-corp-corp:hasDateOfIncorporation rdf:resource="&fibo-be-le-usee;AlphabetIncIncorporationDate"/>
+		<fibo-be-le-cb:isIncorporatedIn rdf:resource="&fibo-be-ge-usj;StateOfDelawareJurisdiction"/>
+		<fibo-fnd-org-fm:isDomiciledIn rdf:resource="&lcc-3166-1;UnitedStatesOfAmerica"/>
+		<fibo-fnd-rel-rel:hasLegalName>Alphabet Inc.</fibo-fnd-rel-rel:hasLegalName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;AlphabetIncBusinessEntityIdentifier">
+		<rdf:type rdf:resource="&fibo-be-corp-corp;RegistrationIdentifier"/>
+		<rdfs:label>business entity identifier for Alphabet Inc.</rdfs:label>
+		<skos:definition>individual representing the Delaware Division of Corporations business entity identifier for Alphabet Inc.</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-be-le-usee;AlphabetInc-US-CA"/>
+		<fibo-fnd-rel-rel:hasTag>5786925</fibo-fnd-rel-rel:hasTag>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;AlphabetIncIncorporationDate">
+		<rdf:type rdf:resource="&fibo-fnd-dt-fd;Date"/>
+		<rdfs:label>Alphabet Inc. incorporation date</rdfs:label>
+		<skos:definition>individual representing the date that Alphabet Inc. was first registered as a corporation in the State of Delaware</skos:definition>
+		<fibo-fnd-dt-fd:hasDateValue>2015-07-23</fibo-fnd-dt-fd:hasDateValue>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;AppleInc-US-CA">
+		<rdf:type rdf:resource="&fibo-be-corp-corp;PubliclyHeldCompany"/>
+		<rdf:type rdf:resource="&fibo-be-le-cb;ForProfitCorporation"/>
+		<rdfs:label>Apple Inc. US-CA</rdfs:label>
+		<skos:definition>individual representing the Apple Inc. legal entity that is a California stock corporation</skos:definition>
+		<fibo-be-corp-corp:hasDateOfIncorporation rdf:resource="&fibo-be-le-usee;AppleIncIncorporationDate"/>
+		<fibo-be-le-cb:isIncorporatedIn rdf:resource="&fibo-be-ge-usj;StateOfCaliforniaJurisdiction"/>
+		<fibo-fnd-org-fm:isDomiciledIn rdf:resource="&lcc-3166-1;UnitedStatesOfAmerica"/>
+		<fibo-fnd-rel-rel:hasLegalName>Apple Inc.</fibo-fnd-rel-rel:hasLegalName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;AppleIncBusinessEntityIdentifier">
+		<rdf:type rdf:resource="&fibo-be-corp-corp;RegistrationIdentifier"/>
+		<rdfs:label>business entity identifier for Apple Inc.</rdfs:label>
+		<rdfs:seeAlso rdf:resource="https://businessfilings.sos.ca.gov/frmDetail.asp?CorpID=00806592"/>
+		<skos:definition>individual representing the California Department of Corporations business entity identifier for Apple Inc.</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-be-le-usee;AppleInc-US-CA"/>
+		<fibo-fnd-rel-rel:hasTag>C0806592</fibo-fnd-rel-rel:hasTag>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-le-usee;AppleIncIncorporationDate">
+		<rdf:type rdf:resource="&fibo-fnd-dt-fd;Date"/>
```

Comment:
The range of the property is Date, not ExplicitDate, so that's what I did in the example.

If you are not happy with that, then please raise an issue against BE, for using explicit date in the range of the property for incorporation date.

Reply:
I'll let the OWL speak for itself...
	<owl:Class rdf:about="&fibo-fnd-dt-fd;ExplicitDate">
		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-fd;Date"/>

Reply:
We have hygiene tests that test for certain things, including crossing subproperty/superproperty hierarchies and types.  My concern is that if I make an explicit date the type, then I should change the property to hasExplicitDate from hasDate, and I did not want to spend the effort for this particular issue.  I was supposed to be creating examples, not revamping the ontology in BE.

Reply:
It has farther reaching implications if I change the property, in other words.

Reply:
I don't see why triples for individuals cannot use a subclass - the restriction and range are still satisfied.

Reply:
Not specific to this PR, let's discuss.